### PR TITLE
Add fixes and updates in some services

### PR DESCRIPTION
**Pull request by rodrigoaasm**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
dojot/dojot#2569

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: storage/time-series/influxdb/retriever/Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN apk add --no-cache --virtual .build-deps \
 
 COPY package-lock.json package-lock.json
 
-RUN npm install --only=prod
+RUN npm install
```
Comment (ademirr) [inline storage/time-series/influxdb/retriever/Dockerfile:28]: Nesse dockerfile não deveria ter o --only=prod?

File: storage/time-series/influxdb/storer/app/influx/DataWriter.js [hunk 2]
```diff
@@ -15,19 +15,22 @@ class DataWriter {
    * @param {String} url   Url to access influxdb
    * @param {String} token  A token with write permission in all orgs
    * @param {String} defaultBucket  Bucket Name for all data write
+   * @param {Boolean} writeAsString
```
Comment (ademirr) [inline storage/time-series/influxdb/storer/app/influx/DataWriter.js:18]: Quando comitei essas alterações dos tipos de dados, esqueci de colocar o comentário no parâmetro writeAsString. Pra ficar parecido com o readAsString, sugiro colocar: Indicates whether to write Influxdb data as a String

**Review by ademirr — CHANGES_REQUESTED** [review]

File: storage/time-series/influxdb/retriever/Dockerfile (hunk 1 above)
Comment (rodrigoaasm) [reply storage/time-series/influxdb/retriever/Dockerfile:28]: Em uma das libs do npm, existe um erro conhecido quando passa esse parâmetro no install.

File: storage/time-series/influxdb/storer/app/influx/DataWriter.js (hunk 2 above)
Comment (rodrigoaasm) [reply storage/time-series/influxdb/storer/app/influx/DataWriter.js:18]: Essa é branch do keycloak, quando a correção for efetuada na development será enviada para cá.

**Review by ademirr — APPROVED** [review]

**Pull request merged**